app/scripts/views/settings.js

```diff
@@ -206,6 +206,9 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
+          // Unset uid otherwise it will henceforth be impossible
+          // to sign in to different accounts inside this tab.
+          self.relier.unset('uid');
           self.user.clearSignedInAccount();
```

Comment:
I think we should just unset `uid`, `email` and `preVerifyToken` for now, the others are used for metrics reporting.